apps/web-giddh/src/app/shared/header/header.component.ts

```diff
@@ -661,6 +661,14 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
                 }
             }
         });
+
+        // for voucher company message
+        this.voucherApiVersion = this.generalService.voucherApiVersion;
```

Comment:
remove this line as it's already defined

Reply:
done